src/components/select/select.js

```diff
@@ -1122,12 +1140,12 @@ function SelectProvider($$interimElementProvider) {
       function watchAsyncLoad() {
         if (opts.loadingAsync && !opts.isRemoved) {
           scope.$$loadingAsyncDone = false;
-          scope.progressMode = 'indeterminate';
+          scope.$mdProgressMode = 'indeterminate';
```

Comment:
In the `SelectMenuDirective` I changed the scope from `true` to `false`. Adding the `$md` insures the variable does not get clobbered by angular I believe? @jelbourn for clarification?